src/mixins/allowable-actions-mixin.ts

```diff
@@ -16,6 +16,33 @@ export default class AllowableActionsMixin extends Vue {
   @Getter(useBusinessStore) isGoodStanding!: boolean
   @Getter(useRootStore) isRoleStaff!: boolean
   @Getter(useRootStore) getUserInfo!: any
+  @Getter(useFilingHistoryListStore) getRegistrationFiling!: any
+
+  /**
+   * Checks if the user is a self-registered owner/operator.
+   */
+  private get isSelfRegisteredOwnerOperator (): boolean {
+    const filing = this.getRegistrationFiling
+    if (!filing) return false
+
+    const { parties } = filing?.registration as FilingRegistraionIF
+    if (!parties) return false
+
+    const { officer: completingParty } =
+      parties.find(party => party.roles.find(role => role.roleType === Roles.COMPLETING_PARTY))
```

Comment:
Why do you need the completing party of the original registration?

Reply:
The current rule for the DBC is that the registering owner and operator must be the same individual. Not sure if there's a different way to determine this? I've based it off of what I was told by @argush3 but it's entirely possible I have misinterpreted that.

Reply:
I'm not sure what "registering owner" is.

Note that sometimes staff file registrations, and I'm not sure who the completing party would be then. @argush3 ?

Reply:
Well, my understanding is to use DC, you need to be logged in as the actual proprietor via BCSC.  Is this not the case?

If it is the case that you need to be logged in as the proprietor, then all 3(proprietor, completing party and logged in user) names should match when filing the registration.

Is staff scenario actually allowed for DC users?

Reply:
I am wondering whether the completing party is always the same as the proprietor. What if staff, or a lawyer, filed the registration?

Isn't is enough to compare the proprietor with the current user?

(You're right, staff can't access DBC, but they might have filed the original registration.)

Reply:
I don't know the business rules well enough.

I thought to use DBC, the actual proprietor needs to do the registration.  i.e. a staff or a lawyer can't do it for them

Reply:
I'm trying to get an answer for this.

Reply:
Argus is correct. At least for the MVP, all three need to be the same. In later phases this may be relaxed.

Reply:
Our BA says,
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/afaa6f2b-e7f9-44d2-9e96-42d66ae43931)

But OK for MVP.